### PR TITLE
openstack-crowbar: enable cloud7 builds (SCRD-8855)

### PR DESCRIPTION
**Pull request by stefannica**
Update the `openstack-crowbar` core job and ansible scripts to
support deploying cloud 7 in the ECP.

### CODE REVIEW

**Comment by stefannica** [comment]
Tested with: https://ci.suse.de/blue/organizations/jenkins/cloud-crowbar7-job-x86_64/detail/cloud-crowbar7-job-x86_64/10/pipeline

**Comment by viccuad** [comment]
Seems that the backports repo is not available for soc7 sle12sp2: https://ci.suse.de/blue/organizations/jenkins/cloud-crowbar7-job-x86_64/detail/cloud-crowbar7-job-x86_64/10/pipeline/#step-135-log-21.
The only solution that I see left is to download a tar and install it that way.

The HA failure should not be happening, there's no set up of the python backports repo at that point. It seems that both were run on the same ci slot, `cloud-ardana-ci-slot8`.

**Comment by stefannica** [comment]
> Tested with:
> 
> * 2 nodes: https://ci.suse.de/blue/organizations/jenkins/cloud-crowbar7-job-x86_64/detail/cloud-crowbar7-job-x86_64/10/pipeline
> * 4 nodes/HA and [this crowbar-ha patch](https://github.com/crowbar/crowbar-ha/pull/355): https://ci.suse.de/blue/organizations/jenkins/cloud-crowbar7-job-ha-x86_64/detail/cloud-crowbar7-job-ha-x86_64/7/pipeline

@viccuad , those jobs both failed due to the missing python backports repository. I had to switch to using rpm packages for all our tempest JUnit dependencies, and resort to using the official Leap repositories, which are better aligned to the SLE ones.

New jobs to test this change:
* 2 nodes: https://ci.suse.de/blue/organizations/jenkins/cloud-crowbar7-job-x86_64/detail/cloud-crowbar7-job-x86_64/12/pipeline
* 4 nodes: https://ci.suse.de/blue/organizations/jenkins/cloud-crowbar7-job-ha-x86_64/detail/cloud-crowbar7-job-ha-x86_64/9/pipeline

File: scripts/jenkins/ardana/ansible/run-crowbar-tests.yml [hunk 1]
```diff
@@ -71,14 +71,9 @@
...
       always:
-        - name: Ensure repo for python-virtualenv is removed
-          zypper_repository:
-            name: python_backports
-            state: absent
-            runrefresh: yes
 
         - name: Generate xml subunit results
-          shell: "{{ subunit_tools_venv }}/bin/subunit2junitxml {{ tempest_results_subunit }} > {{ subunit_xml_results }}"
+          shell: "/usr/bin/subunit2junitxml {{ tempest_results_subunit }} > {{ subunit_xml_results }}"
```
[the remark this answers is not in the document]
Comment (stefannica) [reply scripts/jenkins/ardana/ansible/run-crowbar-tests.yml]: > Seems all good :). Minor nitpicking, why not call the one in the path directly?

Done.

**Pull request merged**